tests/python/unittest/test_image.py

```diff
@@ -92,6 +92,26 @@ def test_imdecode(self):
             cv_image = cv2.imread(img)
             assert_almost_equal(image.asnumpy(), cv_image)
 
+    def test_imdecode_bytearray(self):
+        try:
+            import cv2
+        except ImportError:
+            return
+        for img in TestImage.IMAGES:
+            with open(img, 'rb') as fp:
+                str_image = bytearray(fp.read())
+                image = mx.image.imdecode(str_image, to_rgb=0)
+            cv_image = cv2.imread(img)
+            assert_almost_equal(image.asnumpy(), cv_image)
+
+    @raises(ValueError)
+    def test_imdecode_empty_buffer(self):
+        try:
```

Comment:
import cv2 is not necessary here right? Opencv is internally linked to mxnet library in C++ backend.

Reply:
I actually don't know, I just copied from test case: test_imdecode(self)
And I have no idea why we catch ImportError exception.
Let me know if you want to remove the import.

Reply:
I think it can be removed for test_imdecode_empty_buffer

Reply:
OK, removed.